### PR TITLE
Transcript lengths for DESeq2

### PR DESCRIPTION
**Pull request by pinin4fjords**
This PR adds a much demanded feature- usage of transcript length matrices from tximport when using the outputs of the nf-core rna-seq workflow.

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/differentialabundance/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/differentialabundance _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by pinin4fjords** [comment]
@nf-core-bot fix linting

File: modules.json [hunk 1]
```diff
@@ -36,8 +36,8 @@
                         "installed_by": ["modules"]
                     },
                     "deseq2/differential": {
-                        "branch": "master",
-                        "git_sha": "25047aa940979f64b31d19b94c483a9254263892",
+                        "branch": "deseq2_transcript_lengths",
```
Comment (maxulysse) [inline modules.json:39]: Why not from master?

**Review by maxulysse — APPROVED** [review]

File: modules.json (hunk 1 above)
Comment (pinin4fjords) [reply modules.json:39]: It will be, just wanted to test while we solved the issues in https://github.com/nf-core/modules/pull/4376

**Comment by pinin4fjords** [comment]
Thanks @maxulysse !

**Pull request merged**